TOC.md

```diff
@@ -52,14 +52,14 @@
   + 扩容缩容
     - [使用 Ansible 扩容缩容](dev/how-to/scale/with-ansible.md)
   + 升级
-    - [升级至 TiDB 3.0](dev/how-to/upgrade/to-tidb-3.0.md)
+    - [升级至 TiDB 3.0](dev/how-to/upgrade/from-previous-version.md)
```

Comment:
This renaming affects the link in [this file](https://github.com/pingcap/docs-cn/blob/master/dev/how-to/upgrade/rolling-updates-with-ansible.md). Please update it.